src/win/util.c

```diff
@@ -1230,3 +1230,47 @@ int uv_os_homedir(char* buffer, size_t* size) {
   *size = bufsize - 1;
   return 0;
 }
+
+
+int uv_os_tmpdir(char* buffer, size_t* size) {
+  wchar_t path[MAX_PATH];
+  DWORD bufsize;
+  size_t len;
+
+  if (buffer == NULL || size == NULL || *size == 0)
+    return UV_EINVAL;
+
+  len = GetTempPathW(MAX_PATH, path);
+
+  if (len == 0) {
+    return uv_translate_sys_error(GetLastError());
+  } else if (len > MAX_PATH) {
+    /* This should not be possible */
+    return UV_EIO;
+  }
+
+  /* The returned directory should not have a trailing slash, unless it */
+  /* points at a drive root, like c:\. Remove it if needed.*/
+  if (path[len - 1] == L'\\' &&
+      !(len == 3 && path[1] == L':')) {
+    len--;
+    path[len] = L'\0';
+  }
+
+  /* Check how much space we need */
+  bufsize = uv_utf16_to_utf8(path, -1, NULL, 0);
```

Comment:
Can you replace these with the actual Windows API call? It's a one liner anyway and @piscisaureus and I have stopped using them for a while. At some point that function will disappear altogether.